pydarn/utils/conversions.py

```diff
@@ -81,7 +81,26 @@ def dmap2dict(dmap_records: List[dict]) -> List[dict]:
         dmap_list.append(OrderedDict(dmap_dict))
     return dmap_list
 
+def gateGroundScatter(y, Re, hgt=250):
```

Comment:
At the top of this file add your copyright for this function please :)

Reply:
Hey @mts299 Do you have any example of what I should put in the copyright for the function?